code/game/objects/structures/lattice.dm

```diff
@@ -55,6 +55,27 @@
 	if(current_size >= STAGE_FOUR)
 		deconstruct()
 
+/obj/structure/lattice/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
+	switch(the_rcd.mode)
+		if(RCD_FLOORWALL)
+			return list("mode" = RCD_FLOORWALL, "delay" = 0, "cost" = 1)
```

Comment:
Cost is the same as using the RCD on a space turf that has a lattice/catwalk placed on top.

Reply:
Call underlying space rcd_vals then instead of duplicating the values.

Reply:
How do I call the rcd_vals of /turf/open/space? Sorry, I'm a bit new to this.

Reply:
Get the turf with `get_turf(lattice)` then just call rcd_vals(user,the_rcd) on it. Make sure it's space, you can pass the action through in the effect call too.